src/pytorch_ood/detector/__init__.py

```diff
@@ -156,3 +156,4 @@
 from .tscaling import TemperatureScaling
 from .vim import ViM
 from .webo import WeightedEBO
+from .distance2clusters import Distance2Clusters
```

Comment:
In this file, you will have to add Distance2Clusters to the comments above, otherwise, it will not show up in the documentation.